.ember-cli.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+process.env.EMBER_VERSION = "OCTANE";
```

Comment:
we aren't moving to octane yet 🙂

Reply:
Ah ok.. I hadn't seen that note about using glimmer components in an addon until just now. I had tried `ember feature:enable template-only-glimmer-components` after the upgrade and it added the feature to the `optional-features.json` but still wasn't generating glimmer components so I thought we needed this 😝I'll update